docs/snippets/providers/jiraonprem-snippet-autogenerated.mdx

```diff
@@ -0,0 +1,51 @@
+{/* This snippet is automatically generated using scripts/docs_render_provider_snippets.py 
+Do not edit it manually, as it will be overwritten */}
+
+## Authentication
+This provider requires authentication.
+- **host**: Jira Host (required: True, sensitive: False)
+- **personal_access_token**: Jira PAT (required: True, sensitive: True)
+
+
+## In workflows
+
+This provider can be used in workflows.
+
+
+As "step" to query data, example:
+```yaml
+steps:
+    - name: Query jiraonprem
+      provider: jiraonprem
+      config: "{{ provider.my_provider_name }}"
+      with:
+        ticket_id: {value}  
+        board_id: {value}  
+        kwargs: {value}  # The providers with context
```

Comment:
what does providers with context mean?

Reply:
I don't know, it's a comment from the provider code.